### ISSUE TITLE
no such file to load -- ninesixty

### ISSUE THREAD

**Issue by pgorley** [nextmat/compass-960-plugin#30]
I see that other people have had this issue on ubuntu, but I haven't seen a clear way to resolve it.

Basically, when I run compass watch, it throws back a fun error message:
LoadError on line 105 of /usr/lib/ruby/1.8/compass/configuration/data.rb: no such file to load -- ninesixty
Run with --trace to see the full backtrace

So, if anyone has any suggestions on how to magically get 960 to work in my environment, I would be eternally grateful.

pgorley@gut1:/var/www/wgu$ ruby -v
ruby 1.8.7 (2010-08-16 patchlevel 302) [x86_64-linux]

pgorley@gut1:/var/www/wgu$ gem -v
1.3.7

pgorley@gut1:/var/www/wgu$ compass version
Compass 0.10.6
Copyright (c) 2008-2009 Chris Eppstein
Released under the MIT License.

pgorley@gut1:/var/www/wgu$ cat config.rb
require 'ninesixty'
# Require any additional compass plugins here.
# Set this to the root of your project when deployed:

http_path = "/"
css_dir = "css"
sass_dir = "src"
images_dir = "images"
javascripts_dir = "js"
# To enable relative paths to assets via compass helper functions. Uncomment:
# relative_assets = true

pgorley@gut1:/var/www/wgu$ compass --list frameworks
WARNING: This interface is deprecated. Please use the new subcommand interface.
See `compass help` for more information.

Available Frameworks & Patterns:
- blueprint
  - blueprint/semantic   - The blueprint framework for use with semantic markup.
  - blueprint/link_icons - Icons for common types of links
  - blueprint/basic      - A basic blueprint install that mimics the actual blueprint css.
  - blueprint/buttons    - Button Plugin
  - blueprint/project    - The blueprint framework.
- compass
  - compass/ellipsis     - Plugin for cross-browser ellipsis truncated text.
  - compass/extension    - Generate a compass extension.
  - compass/project      - The default project layout.

Cheers!

**Comment by pgorley** [comment]
Wow, github really jacked up the formatting.  I'm sorry that it looks so wonky.

**Comment by nextmat** [comment]
Thanks for being thorough on the details. I haven't seen this issue before. Can you confirm that rubygems is already loaded before you get to the `require` line and that the gem is installed on your system?

**Comment by pgorley** [comment]
Yes. rubygems was installed prior to setting the require.

I honestly didn't know how to fix this, so I decided to build another box and see if I could replicate the issue.  This new box is running Ubuntu 11.10 server, rather than 11.04 server.  The 11.10 box...works.  I made sure that all of the gem stuff was installed using sudo.  So, I suppose I can close this issue unless someone really cares to fix it for 11.04LTS.
